src/helpers/configHelpers.ts

```diff
@@ -43,19 +83,20 @@ export function setMessage(guild: Guild, messageKey: MessageKey, value: string):
 	return setConfig(guild, config);
 }
 
-export function enableAutothreading(guild: Guild, channelId: string, message = ""): boolean {
+export function enableAutothreading(guild: Guild, channelId: string, includeBots?: boolean, archiveImmediately?: boolean, messageContent?: string): boolean {
 	const config = getConfig(guild.id);
 	if (!config || !config.threadChannels) { return false; }
-	if (message.length > 2000) { return false; }
+	if ((messageContent?.length ?? 0) > 2000) { return false; }
```

Comment:
```suggestion
	if ((messageContent?.length ?? 0) > 2000) return false;
```

Reply:
Ugh this is inconsistent everywhere 😅 I'll add a config for prettier eventually and just deal with this once and for all when that happens. Hoping to integrate more DX stuff before v3!